.zap/rules.tsv

```diff
@@ -0,0 +1,149 @@
+40016	WARN	(Cross Site Scripting (Persistent) - Prime - Active/release)
+40017	WARN	(Cross Site Scripting (Persistent) - Spider - Active/release)
+40018	WARN	(SQL Injection - Active/release)
+40019	WARN	(SQL Injection - MySQL - Active/release)
+40020	WARN	(SQL Injection - Hypersonic SQL - Active/release)
+40021	WARN	(SQL Injection - Oracle - Active/release)
+40022	WARN	(SQL Injection - PostgreSQL - Active/release)
+40023	WARN	(Possible Username Enumeration - Active/beta)
+40024	WARN	(SQL Injection - SQLite - Active/release)
+40025	WARN	(Proxy Disclosure - Active/beta)
+40026	WARN	(Cross Site Scripting (DOM Based) - Active/release)
+40027	WARN	(SQL Injection - MsSQL - Active/release)
+40028	WARN	(ELMAH Information Leak - Active/release)
+40029	WARN	(Trace.axd Information Leak - Active/release)
+40031	WARN	(Out of Band XSS - Active/beta)
+40032	WARN	(.htaccess Information Leak - Active/release)
+40033	WARN	(NoSQL Injection - MongoDB - Active/alpha)
+40034	WARN	(.env Information Leak - Active/release)
+40035	WARN	(Hidden File Finder - Active/release)
+40038	WARN	(Bypassing 403 - Active/beta)
+40039	WARN	(Web Cache Deception - Active/alpha)
+40040	WARN	(CORS Header - Active/beta)
+40042	WARN	(Spring Actuator Information Leak - Active/release)
+40043	WARN	(Log4Shell - Active/release)
+40044	WARN	(Exponential Entity Expansion (Billion Laughs Attack) - Active/beta)
+40045	WARN	(Spring4Shell - Active/release)
+40046	WARN	(Server Side Request Forgery - Active/beta)
+40047	WARN	(Text4shell (CVE-2022-42889) - Active/beta)
+41	WARN	(Source Code Disclosure - Git - Active/beta)
+42	WARN	(Source Code Disclosure - SVN - Active/beta)
+43	WARN	(Source Code Disclosure - File Inclusion - Active/beta)
+50000	WARN	(Script Active Scan Rules - Active/release)
+50001	WARN	(Script Passive Scan Rules - Passive/release)
+6	WARN	(Path Traversal - Active/release)
+7	WARN	(Remote File Inclusion - Active/release)
+90001	WARN	(Insecure JSF ViewState - Passive/release)
+90002	WARN	(Java Serialization Object - Passive/beta)
+90003	WARN	(Sub Resource Integrity Attribute Missing - Passive/beta)
+90004	WARN	(Insufficient Site Isolation Against Spectre Vulnerability - Passive/beta)
+90005	WARN	(Fetch Metadata Request Headers - Passive/alpha)
+90011	WARN	(Charset Mismatch - Passive/release)
+90017	WARN	(XSLT Injection - Active/release)
+90019	WARN	(Server Side Code Injection - Active/release)
+90020	WARN	(Remote OS Command Injection - Active/release)
+90021	WARN	(XPath Injection - Active/release)
+90022	WARN	(Application Error Disclosure - Passive/release)
+90023	WARN	(XML External Entity Attack - Active/release)
+90024	WARN	(Generic Padding Oracle - Active/release)
+90025	WARN	(Expression Language Injection - Active/beta)
+90026	WARN	(SOAP Action Spoofing - Active/beta)
+90027	WARN	(Cookie Slack Detector - Active/beta)
+90028	WARN	(Insecure HTTP Method - Active/beta)
+90029	WARN	(SOAP XML Injection - Active/beta)
+90030	WARN	(WSDL File Detection - Passive/beta)
```

Comment:
```suggestion
90030	IGNORE	(WSDL File Detection - Passive/beta)
```
In the current test case the test is a false positive, the API returns a 400 code.
As part of a SOAP integration, it can be possible to supply a WSDL